tests/foreman/ui/test_host.py

```diff
@@ -2143,6 +2143,121 @@ def test_bootc_booted_container_images(target_sat, bootc_host, function_ak_with_
         assert booted_container_image_info[1]['Hosts'] == '1'
 
 
+@pytest.mark.tier2
+def test_bootc_host_details(target_sat, bootc_host, function_ak_with_cv, function_org):
+    """Create a bootc host, and read it's information via the Host Details UI
+
+    :id: 842356e9-8798-421d-aca6-0a1774c3f22b
+
+    :expectedresults: Host Details UI contains the proper information for a bootc host
+
+    :CaseComponent:Hosts-Content
+
+    :Verifies:SAT-27171
+
+    :Team: Phoenix-subscriptions
+    """
+    bootc_dummy_info = json.loads(DUMMY_BOOTC_FACTS)
+    assert bootc_host.register(function_org, None, function_ak_with_cv.name, target_sat).status == 0
+    assert bootc_host.subscribed
+
+    with target_sat.ui_session() as session:
+        session.organization.select(function_org.name)
+        values = session.host_new.get_details(bootc_host.hostname, widget_names='details.bootc')
+        assert (
+            values['details']['bootc']['details']['running_image']
+            == bootc_dummy_info['bootc.booted.image']
+        )
+        assert (
+            values['details']['bootc']['details']['running_image_digest']
+            == bootc_dummy_info['bootc.booted.digest']
+        )
+        assert (
+            values['details']['bootc']['details']['rollback_image']
+            == bootc_dummy_info['bootc.rollback.image']
+        )
+        assert (
+            values['details']['bootc']['details']['rollback_image_digest']
+            == bootc_dummy_info['bootc.rollback.digest']
+        )
+
+
+def test_bootc_rex_job(target_sat, bootc_host, function_ak_with_cv, function_org):
+    """Run all bootc rex job (switch, upgrade, rollback, status) through Host Detais UI
+
+    :id: ef92a5f7-8cc7-4849-822c-90ea68b10554
+
+    :expectedresults: Host Details UI links to the proper template, which runs successfully for all templates
+
+    :CaseComponent: Hosts-Content
+
+    :Verifies:SAT-27154, SAT-27158
+
+    :Team: Phoenix-subscriptions
+    """
+    BOOTC_SWITCH_TARGET = "images.paas.redhat.com/bootc/rhel-bootc:latest-10.0"
+    BOOTC_BASE_IMAGE = "localhost/tpl-bootc-rhel-10.0:latest"
```

Comment:
I would recommend adding these to the config even if it would mean creating a new config file with a new `BOOTC` or `CHOOSE_THE_BEST_NAME` section.
Recently, we have moved container-related constants to a config in #17690.

Reply:
Similar to above, I'll clean this up in the 6.18 work.